warema_wms/wms_controller.py

```diff
@@ -18,16 +18,17 @@
 RX_SHADE_STATE = '0431'
 
 TX_MOVE_SHADE = '0821'
-SHADE_POSITION = '03{}ffffff'
+SHADE_POSITION = '03{}dfffff'   # df = trial to move raffstores in tilt closed pos., working in console but not in HA
+TILT_POSITION = '03ff{}ffff'
```

Comment:
This is surprising to me. I would have guessed that position and tilt should be send in one command. The way you set it up is with two commands. One for position where the tilt is always `ff` and one for tilt where the position is always `ff`.
If that's how the warema webcontrol does it, it's probably correct, but as I said, I have no way of verifying it.

Reply:
If this is how it is done by the WebControl App, than it is fine like this.

Reply:
sorry, I probably expressed myself misleadingly, I think you're right, it should only be one command.
I just didn't have it on my radar because I just "copied" the behavior of the venetian blinds as they do when there triggered via web control (first move to position, then set tilt)
Unfortunately I didn't have time to look at it yesterday.
Sorry for the confusion.

Basically, my solution should be good for HA (first move into position, then set tilt), but it's not technically clean

Reply:
In this case you should definitely try out if this even works as one command before changing it. Could be that the webcontrol server only accepts the two commands separately (as it isn't the best implementation from what I can gather).

Reply:
will do my best and come back with results